ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -160,9 +159,9 @@ public void shouldRunThePriorCommandsCorrectly() {
 
     // Then:
     final InOrder inOrder = inOrder(statementExecutor);
-    inOrder.verify(statementExecutor).handleRestore(eq(queuedCommand1));
-    inOrder.verify(statementExecutor).handleRestore(eq(queuedCommand2));
-    inOrder.verify(statementExecutor).handleRestore(eq(queuedCommand3));
+    inOrder.verify(statementExecutor).handleRestore(eq(queuedCommand1), any());
```

Comment:
I'm guessing you didn't mean to keep all of these? This PR doesn't change the `handleRestore` syntax anymore (same throughout the PR)